test/package.js

```diff
@@ -40,195 +42,204 @@ themeFiles = [
 	"images/ui-icons*png"
 ];
 THEME_FILES_TESTCASES = themeFiles.length;
-function themeFilesCheck( test, files, theme ) {
+function themeFilesCheck( assert, files, theme ) {
 	themeFiles.forEach( function( filepath ) {
 		if ( theme ) {
-			test.ok( filePresent( files, filepath ), "Missing a theme file \"" + filepath + "\"." );
+			assert.ok( filePresent( files, filepath ), "A theme file \"" + filepath + "\" present." );
 		} else {
-			test.ok( !filePresent( files, filepath ), "Should not include the theme file \"" + filepath + "\"." );
+			assert.ok( !filePresent( files, filepath ), "The theme file \"" + filepath + "\" not included." );
 		}
 	} );
 }
 
-tests = {
-	"test: select all components": {
-		"with the default theme": function( test ) {
-			var pkg = new Packager( this.files, Package, {
-				components: this.allComponents,
-				themeVars: defaultTheme
-			} );
-			test.expect( COMMON_FILES_TESTCASES + THEME_FILES_TESTCASES );
-			pkg.toJson( function( error, files ) {
-				if ( error ) {
-					return test.done( error );
-				}
-				commonFilesCheck( test, files );
-				themeFilesCheck( test, files, true );
-				test.done();
-			} );
-		},
-		"with a different theme": function( test ) {
-			var pkg = new Packager( this.files, Package, {
-				components: this.allComponents,
-				themeVars: themeGallery[ 1 ].vars
-			} );
-			test.expect( COMMON_FILES_TESTCASES + THEME_FILES_TESTCASES );
-			pkg.toJson( function( error, files ) {
-				if ( error ) {
-					return test.done( error );
-				}
-				commonFilesCheck( test, files );
-				themeFilesCheck( test, files, true );
-				test.done();
-			} );
-		}
-	},
-	"test: select all widgets": function( test ) {
-		var allWidgets = this.allWidgets;
-		var pkg = new Packager( this.files, Package, {
+function runTests( context, jQueryUiVersion ) {
+
+	QUnit.test( `[${ jQueryUiVersion }]: select all components with the default theme`, function( assert ) {
+		assert.expect( COMMON_FILES_TESTCASES + THEME_FILES_TESTCASES );
+
+		const done = assert.async();
+
+		const pkg = new Packager( context.files, Package, {
+			components: context.allComponents,
+			themeVars: defaultTheme
+		} );
+		pkg.toJson( function( error, files ) {
+			if ( error ) {
+				return done( error );
```

Comment:
`done()` doesn't take any parameters. To reject tests with an error, you may want to promisify and return (or await) the promisified result instead.

Reply:
@Krinkle Thanks for the catch! Updated.